src/screens/SearchResultsScreen.tsx

```diff
@@ -153,13 +154,14 @@ const SearchResultsScreen: React.FC = (): JSX.Element => {
     }
 
     // TODO: #438 Handle this error better
-    if (!showDetails) {
-        return (
-            <>
-                {SearchResultHeader}
-                <Typography variant='h6'>Sorry! No show data...</Typography>
-            </>
-        );
+    if (showDetails) {
+        if (showDetails.length === 0) {
```

Comment:
nit:
```suggestion
    if (showDetails && showDetails.length === 0) {
```

Reply:
I recall a `possibly null` TS error, but can't seem to replicate.

Reply:
Just to clarify, these code snippets are the same. I just think my suggestion is more concise. It will still cover the null check.